charms/worker/k8s/src/charm.py

```diff
@@ -346,14 +341,14 @@ def _create_cos_tokens(self):
             self.distributor.allocate_tokens(relation=rel, token_strategy=TokenStrategy.COS)
 
     @on_error(
-        WaitingStatus("Waiting for enable functionalities"),
+        WaitingStatus("Waiting to enable Functionalities"),
```

Comment:
We are now using ✨ `features` ✨ 
```suggestion
        WaitingStatus("Waiting to enable features"),
```